src/utils/dateUtils.js

```diff
@@ -1,10 +1,6 @@
-export const getToday = () => {
-  const today = new Date();
-  today.setHours(12, 0, 0, 0);
-  return today;
-};
-
 /**
+ * Takes two dates and creates an array of all dates
+ * between them, inclusive.
  *
  * @param {Object} startDate
```

Comment:
Is `Object` correct? - shouldn't it be `Date`?

Reply:
sure